src/libsystemd-network/radv-internal.h

```diff
@@ -71,7 +71,7 @@
 /* rfc6275 7.4 Neighbor Discovery Home Agent Lifetime.
  * The default value is the same as the Router Lifetime
  * The maximum value corresponds to 18.2 hours. value of 0 MUST NOT be used.*/
-#define RADV_MAX_HOME_AGENT_LIFETIME_USEC (65535 * USEC_PER_SEC)
+#define RADV_MAX_HOME_AGENT_LIFETIME_USEC (65535ULL * USEC_PER_SEC)
```

Comment:
Is this really necessary?

Reply:
Yeah, without it the build complains on x86_64:
```
In file included from ../src/basic/macro.h:455,
                 from ../src/basic/alloc-util.h:10,
                 from ../src/basic/hash-funcs.h:4,
                 from ../src/basic/hashmap.h:8,
                 from ../src/shared/dns-domain.h:10,
                 from ../src/network/networkd-radv.c:9:
../src/network/networkd-radv.c: In function ‘config_parse_router_home_agent_lifetime’:
../src/network/networkd-radv.c:1626:28: error: format ‘%llu’ expects argument of type ‘long long unsigned int’, but argument 11 has type ‘long unsigned int’ [-Werror=format=]
 1626 |                            "Invalid %s= must be in the range 1...%llu seconds, ignoring: %s", lvalue,
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/basic/log.h:361:125: note: in definition of macro ‘log_syntax’
  361 |                         ? log_syntax_internal(unit, _level, config_file, config_line, _e, PROJECT_FILE, __LINE__, __func__, __VA_ARGS__) \
      |                                                                                                                             ^~~~~~~~~~~
../src/network/networkd-radv.c:1626:69: note: format string is defined here
 1626 |                            "Invalid %s= must be in the range 1...%llu seconds, ignoring: %s", lvalue,
      |                                                                  ~~~^
      |                                                                     |
      |                                                                     long long unsigned int
      |                                                                  %lu
cc1: some warnings being treated as errors
```

But it should no longer be necessary with `FORMAT_TIMESPAN()`.